schemas/verifiableCredential.js

```diff
@@ -0,0 +1,56 @@
+/*!
+ * Copyright (c) 2012-2022 Digital Bazaar, Inc. All rights reserved.
+ */
+import {extend as _extend} from '../lib/helpers.js';
+import proof from './proof.js';
+import identifier from './identifier.js';
+import {klona} from 'klona';
+import w3cDateTime from './w3cDateTime.js';
+
+// https://www.w3.org/TR/vc-data-model/
+// Based off of the VC Data Model 1.1
+const schema = {
+  type: 'object',
+  title: 'Verifiable Credential',
+  additionalProperties: true,
+  properties: {
+    '@context': {
+      type: 'array',
+      minItems: 1,
+      items: [
+        {type: 'string', const: 'https://www.w3.org/2018/credentials/v1'},
+        {anyOf: [{type: 'string'}, {type: 'object'}]}]
+    },
+    credentialSubject: {
+      anyOf: [
+        {type: 'object'},
+        {type: 'array', minItems: 1, items: {type: 'object'}}
+      ]
+    },
+    id: identifier(),
+    issuer: identifier(),
+    issuanceDate: w3cDateTime(),
+    proof: proof(),
+    type: {
+      type: 'array',
+      minItems: 1,
+      items: {
```

Comment:
The first item has to be `VerifiableCredential`.

Reply:
Addressed here: https://github.com/digitalbazaar/bedrock-validation/pull/44/commits/f6ee4b99d158a51d4af4f9e80985df9dff0014fb